src/App/Expression.cpp

```diff
@@ -2892,10 +2892,18 @@ bool VariableExpression::_renameObjectIdentifier(
     auto it = paths.find(oldPath);
     if (it != paths.end()) {
         v.aboutToChange();
+        const bool originalHasDocumentObjectName = var.hasDocumentObjectName();
+        ObjectIdentifier::String originalDocumentObjectName = var.getDocumentObjectName();
+        std::string originalSubObjectName = var.getSubObjectName();
         if(path.getOwner())
             var = it->second.relativeTo(path);
         else
             var = it->second;
+        if (originalHasDocumentObjectName) {
+            var.setDocumentObjectName(std::move(originalDocumentObjectName),
+                                      true,
+                                      std::move(originalSubObjectName));
```

Comment:
It's not a particularly important thing to fix, but the `std::move` of `originalSubObjectName` won't actually do a move here, because it's a `std::string`, not a `ObjectIdentifier::String` -- that means it's actually getting passed as a `const std::string &` to the constructor of `ObjectIdentifier::String`, which does not do a move, and ultimately copies the string.